src/BuildCheck.UnitTests/Microsoft.Build.BuildCheck.UnitTests.csproj

```diff
@@ -46,4 +50,10 @@
     </None>
   </ItemGroup>
 
+  <Target Name="MSBuildPackagesForTests" AfterTargets="Build">
```

Comment:
This feels as possibly expensive target - can it have inputs and outputs defined to allow for incrementality?

Reply:
please let me know if I got you right with my last change

Reply:
Oh perfect! Thank you!